lesson02/assignment/css/_settings.variables.scss

```diff
@@ -1,12 +1,16 @@
 // Colors
-$gray: #f6f5f6;
-$white: #ffffff;
-$black: #000000;
-$green: #7acb00;
-$pink: #ff0097;
-$red: #F1352B;
-$orange: #ff6600;
-$yellow: #ffb617;
-$robins: #01c1d6;
-$purple: #6e27c5;
-$blue: #150f96;
\ No newline at end of file
+$charcoal: #707070;
+$light-blue: #F2F4F7;
+$light-grey: #F5F5F5;
+$white: #FFFFFF;
+$faded-blue: #5D8AA8;
+$border: #D8D8D8;
+
+// Fonts
+$font-fam: "Helvetica Neue", Helvetica, Arial, sans-serif;
+
+// Padding
+$padding: 1em;
+
+// Width
+$width: 100%;
```

Comment:
Good stuff Lauren, just need a sass mixin or function for the last point. You could alter the padding/width variables into mixins that take values and return that as padding/width in em or %.